adapters/repos/db/lsmkv/segment_group_compaction.go

```diff
@@ -248,38 +247,21 @@ func (sg *SegmentGroup) stripTmpExtension(oldPath string) (string, error) {
 	return newPath, nil
 }
 
-func (sg *SegmentGroup) initCompactionCycle(interval time.Duration) {
-	if interval == 0 {
-		return
-	}
+func (sg *SegmentGroup) initCompactionCycle() {
```

Comment:
Same as above with `initFlushCycle`, I think the `init` part no longer matches what this method does.